### ISSUE TITLE
`F841` false negative on assignment to multiple variables

### ISSUE THREAD

**Issue by DetachHead** [astral-sh/ruff#8441]
```py
def foo():
    a = bar() # error: F841
    b, c = bar() # no error

def bar():
    ...
```
[playground](https://play.ruff.rs/e10d4570-4368-4d83-af88-57b221fe899f)

**Comment by qdegraaf** [comment]
Could this be intentional? From [the tests](https://github.com/astral-sh/ruff/blob/d350ede992ae30067486e56852ca106675ddd57d/crates/ruff_linter/resources/test/fixtures/pyflakes/F841_1.py#L1C1-L2C45):

```python
def f(tup):
    x, y = tup  # this does NOT trigger F841
```

Unsure if that was left there as a TODO or consciously omitted. @charliermarsh could you clarify?

**Comment by charliermarsh** [comment]
Yeah this is intentional for parity with Flake8. I think it's reasonable. An improvement could be to flag `x` and `y` if _both_ are unused.

**Comment by DetachHead** [comment]
I don't get why it should be treated differently. If it's for the case where you only want to use one of the values from the tulle, you can just prefix it with an `_` to mark it as intentionally unused:
```py
a, _b = foo()
```

**Comment by zanieb** [comment]
Best practice is really

```
a, _ = foo()
```

I don't see why we wouldn't raise a violation here, other than flake8 compatibility

**Comment by charliermarsh** [comment]
I'm not opposed to changing it. I didn't feel strongly so erred towards compatibility.
